src/main/java/codeu/model/data/Activity.java

```diff
@@ -31,6 +31,13 @@
 	private final UUID userId;
 	private final String username;
 	
+	/** 
+	 * Declares whether it is an activity object for a conversation or for a user registration 
+	 * 'C' is used for a conversation while 'R' is used for a registration. 
+	 * May eventually be changed to an enum.
```

Comment:
This definitely should be be an enum. OK to leave this for now but definitely should fix in your next iteration.